go.mod

```diff
@@ -35,4 +35,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 )
 
+replace github.com/codeready-toolchain/api => github.com/xcoulon/api v0.0.0-20211125125708-0f5ec2774aa7
```

Comment:
reminder

Reply:
taken care of in https://github.com/codeready-toolchain/toolchain-e2e/pull/414/commits/7418beab7466332440143f7a3d0f3f937691dcdc